src/js/player.js

```diff
@@ -2009,6 +2002,24 @@ class Player extends Component {
     return this.cache_;
   }
 
+  /**
+   * Resets the internal cache object.
+   *
+   * Using this function outside the player constructor or reset method may
+   * have unintended side-effects.
+   *
+   * @private
+   */
+  resetCache_() {
+    this.cache_ = {
+      lastVolume: 1,
+      lastPlaybackRate: this.defaultPlaybackRate(),
```

Comment:
Should we set `currentTime`, `duration`, `selectedLanguage` to null/empty string here too?

Reply:
I left those out because it appeared they got set in their getter/setters, but it probably makes sense to set a default for everything here.

Reply:
Turns out setting `selectedLanguage` in here has all kinds of side effects. Probably best left to another PR.